sda-commons-server-auth/src/main/java/org/sdase/commons/server/opa/OpaBundle.java

```diff
@@ -181,15 +187,11 @@ private Client createClient(
       clientConfig.setTimeout(Duration.milliseconds(config.getReadTimeout()));
     }
 
-    Client client =
-        new JerseyClientBuilder(environment)
-            .using(clientConfig)
-            .using(objectMapper)
-            .build("opaClient");
+    JerseyClientBuilder jerseyClientBuilder = new JerseyClientBuilder(environment);
+    jerseyClientBuilder.setApacheHttpClientBuilder(
+        new TracedHttpClientInitialBuilder(environment).usingTelemetryInstance(openTelemetry));
```

Comment:
I think I've seen that codesnippet a couple of times now. Does it make sense to extract it to a helper class?

Reply:
yes, I was also surprised to see that the opa bundle and auth bundle are not using clients created from the `sda-commons-client-jersey` and directly using the `JerseyClientBuilder `. I needed to move the dependent class to the `opentelemetry`bundle to reuse it. We could add the dependency, but I thought that there was a reason for that